solr/core/src/java/org/apache/solr/metrics/AggregateMetric.java

```diff
@@ -51,6 +61,18 @@ public String toString() {
           ", updateCount=" + updateCount +
           '}';
     }
+
+    public double toDouble() {
+      if (value instanceof Boolean) {
+        return 0;
+      } 
+      if (!(value instanceof Number)) {
+        log.debug("not a Number: " + value);
```

Comment:
Can you use parametrized logging?

Reply:
yes - if this logging is useful, I may just remove it instead.